README.md

```diff
@@ -1 +1,21 @@
-# elasticsearch-admin-scripts
\ No newline at end of file
+# elasticsearch-admin-scripts
+
+This repo stores the Cerebro team's scripts that make it easier to manage Elasticsearch. Detailed descriptions are in the scripts themselves.
+
+## Example usage
+```bash
+./createIndex.sh --user "username" --env "local" --index "myIndex" --docMappingFile "document_mapping.json"
+```
+
+Every script requires two arguments:
+* `env`: the environment of the cluster the script is run against (e.g. `oscf-dev`)
+* `user`: a user name with access to the cluster in `env`
+
```

Comment:
Might mention how to determine the alias or index names using example commands here, since those are often useful for running other commands.